cmake/std/atdm/common/toss3/environment_new.sh

```diff
@@ -57,7 +57,6 @@ if [ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.2_OPENMPI-2.0.3" ]; then
     export ATDM_CONFIG_SUPERLUDIST_LIBS=${SUPERLUDIST_ROOT}/lib64/libsuperlu_dist.a
     export ATDM_CONFIG_BINUTILS_LIBS="/usr/lib64/libbfd.so;/usr/lib64/libiberty.a"
 
-    export PATH=/usr/tce/packages/gcc/gcc-6.1.0/bin:${PATH}
```

Comment:
Just to be clear, I don't know if that PATH set is needed or not.  I was just surprised that this path was set.  That is all.  Does this break the builds if you remove this?

Reply:
It depends, the build breaks if we remove gcc from PATH for the intel-19 build but not for the intel-18 build.

Reply:
I stand corrected. Removing this path in either intel-1{8,9} config breaks the builds. How does this path point the build system to the required libcxx symbols? Doesn't libcxx reside in the `LD_PRELOAD_PATH`? Or, does the build system need the linker or other executable in this bin directory to find libcxx? I can't tell from inspecting the build system.